setup

```diff
@@ -98,12 +98,26 @@ storage() {
 	touch "${ROOTFS}/var/lib/shared/overlay-images/images.lock" \
 	      "${ROOTFS}/var/lib/shared/overlay-layers/layers.lock"
 
-	sed -e '/additionalimage.*/a "/var/lib/shared",' \
-	    -e 's|^#.*transient_store.*|transient_store=true|g' \
-	    "${ROOTFS}/usr/share/containers/storage.conf" \
-	    > "${ROOTFS}/etc/containers/storage.conf"
-        sed -i "s|^#env = \[\]|env = \[\"TMPDIR=/var/${TMP_QM_IMG_DIR}\"\]|" \
-            "${ROOTFS}/usr/share/containers/containers.conf"
+        sed -e '/^\s*#/d' -e  '/^$/d' \
```

Comment:
OT os not a good idea, it changes the order of configs and subconfigs
Please return to previous format